src/components/PostNote.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+
+interface PostNoteProps {
+  title: string;
+  date: string;
+  description: string;
+  drawOutlineTime?: number; // Time for outline animation (seconds)
+  fadeInTime?: number; // Time for fade-in animation (seconds)
+  animationDelay?: number; // Animation delay time (seconds)
+  width?: number; // Container width
+  height?: number; // Container height
+  variant?: 'default'; // Style variant
+}
+
+const PostNote: React.FC<PostNoteProps> = ({
+  title,
+  date,
+  description,
+  drawOutlineTime = 5,
+  fadeInTime = 5,
+  animationDelay = 3,
+  width = 300,
+  height = width * 1.4,
+}) => {
+  return (
+    <div
+      id="root"
+      className={clsx(
+        'relative flex items-center justify-center mx-auto', // Add centering styles
+      )}
+      style={{ width: `${width}px`, height: `${height}px`, transform: 'translate(8%, 0%)' }}
+    >
+      {/* SVG for outline animation */}
+      <svg
+        className="absolute"
+        width="100%"
+        height="100%"
+        viewBox="0 0 60 84"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        {/* Precise shadow path */}
+        <path
+          className="shadow"
+          d="M52 6 v64 a4 4 0 0 1-4 4 H6 c-1.1 0-2-.9-2-2 v3 c0 2.2 1.8 4 4 4 h42 c4.4 0 8-3.6 8-8 V6 z"
+          fill="#6b7280"
+          style={{
+            opacity: 0,
+            animation: `fadeIn ${fadeInTime}s ease forwards`,
+            animationDelay: `${animationDelay}s`,
+          }}
+        />
+
+        {/* Paper outline */}
+        <path
```

Comment:
It appears there’s a repeated path className="paper-outline" snippet (lines 53 and 54). Double-check if that’s intentional. If one is redundant, remove it to streamline the SVG structure.